ncolony/client/heart.py

```diff
@@ -40,11 +40,11 @@ def makeService():
     """
     configJSON = os.environ.get('NCOLONY_CONFIG')
     if configJSON is None:
-        return
+        return None
```

Comment:
This should not be a part of the diff but it does get it to successfully build.
Once we add "pin-pylint", and rebase, this diff will go away.